CONTRIBUTING.md

```diff
@@ -242,6 +242,10 @@ the Visual C++ build tools so that rust can link against the system c/c++
 libraries. You can see more details on this in the
 [rustup documentation](https://rust-lang.github.io/rustup/installation/windows.html).
 
+Qiskit Terra has a minimum supported Rust version (MSRV) of 1.56.1. This means
+to build Qiskit Terra from source you must have at least rustc version 1.56.1
+installed, older versions will not be able to compile Qiskit.
```

Comment:
Do we need to pin to 1.56.1 exactly in our CI?  Currently we're setting 1.56, but I don't know if that's specific enough.  I guess so, since there's no 1.56.2 release anyway.

Reply:
For CI it shouldn't matter, the rustup toolchain version 1.56 will install the latest version of 1.56 it finds on the index, which is 1.56. It doesn't hurt to be explicit though and just use 1.56.1 everywhere. I'll make this change